source/WrappedText.cpp

```diff
@@ -170,10 +203,6 @@ Point WrappedText::Word::Pos() const
 
 void WrappedText::SetText(const char *it, size_t length)
 {
-	// Clear any previous word-wrapping data. It becomes invalid as soon as the
-	// underlying text buffer changes.
-	words.clear();
```

Comment:
We should probably still clear the words here.

Reply:
Same as the previous comment. I'll revert it.